packages/renderer/src/lib/style/IconsStyle.svelte

```diff
@@ -46,7 +46,7 @@ onMount(() => {
     });
 
     fontsToAdd.forEach(font => {
-      const src = font.src.map(l => `${toUrl(l.location)} format('${l.format}')`).join(', ');
+      const src = font.src.map(l => `${toUrl(l.browserURL)} format('${l.format}')`).join(', ');
```

Comment:
maybe it means we wouldn't the toUrl method ?

Reply:
I thought there was a reason to have `url('file://${location.replace(/'/g, '%27')}')` in the client. Do i move everything in the server?

Reply:
I think that now that we provide the field for the client on the server, we could do everything on the server side so client is dummy, just use the field